src/main/java/com/esri/samples/search/offline_geocode/README.md

```diff
@@ -0,0 +1,43 @@
+<h1>Offline Geocode</h1>
+
+<p>Demonstrates how to perform geocoding and reverse geocoding with offline data.</p>
+
+<p><img src="OfflineGeocode.png"/></p>
+
+<h2>How to use the sample</h2>
+
+<p>For simplicity, the sample comes loaded with a set of addresses. You can select any address to perform geocoding and show it's location on map. To perform reverse geocoding in real-time, click on any location on the ArcGISMap to provide it's address.</p>
```

Comment:
Would add how you can select the marker symbol and move it to reverse geocode on the fly. That was cool and unexpected.